initdb.sql

```diff
@@ -0,0 +1,61 @@
+create table GeneralInfo(
+    base_id int,
+    last_user_id int,
+    last_group_id int,
+    last_doc_id int
+);
+
+insert into GeneralInfo values (0, 0, 0, 0);
+
+create table Users(
+    id int primary key,
+    login text
+);
+
+create table Password(
+    id int primary key,
+    password bytea,
+    constraint fr_user_id foreign key(id) references Users(id)
```

Comment:
Do you really need this field to be primary and foreign key at the same time? Do you really need this Password table at all? Please explain your reasoning behind this decision.

Reply:
The "id" field could be just unique. And it should be a foreign key to correspond to id from Users table. 
Password table could be moved to another database later for security reasons. So this is the main reason to have password and login in separate tables

Reply:
I can see no extra security benefits here, just transactional and computational penalties. BTW, have you ever heard of a salted hash?

Reply:
I think GenerateFromPassword method from bcrypt lib uses salt internally

Reply:
That's right. So your hash strings are already made hard to break. The salt is there to require each password to be attacked individually. You add no extra layer of security by merely placing these strings to a separate table or database, but the risks of data inconsistency are high. Just think about it.